PROTO_tests/tests/sort_test.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pytest
+
+import arkouda as ak
+from arkouda.sorting import SortingAlgorithm
+
+NUMERIC_AND_BIGINT_TYPES = ["int64", "float64", "uint64", "bigint"]
+
+
+def make_ak_arrays(dtype):
+    if dtype in ["int64", "uint64"]:
+        return ak.randint(0, 100, 1000, dtype)
+    elif dtype == "float64":
+        return ak.array(np.random.rand(100) * 10000)
+    elif dtype == "bigint":
+        return ak.randint(0, 100, 1000, dtype=ak.uint64)
+    return None
+
+
+class TestSort:
+    @pytest.mark.parametrize("algo", SortingAlgorithm)
+    @pytest.mark.parametrize("dtype", NUMERIC_AND_BIGINT_TYPES)
+    def test_sort_dtype(self, algo, dtype):
+        pda = make_ak_arrays(dtype)
+
+        if dtype != "bigint":
+            assert ak.is_sorted(ak.sort(pda, algo))
```

Comment:
If we are going to use `ak.is_sorted` to validate `ak.sort`, we need to validate `ak.is_sorted` is functioning properly first. A test should be added for `ak.is_sorted` or after this assertion we should validate that the result is seen as sorted by NumPy. Personally, I am leaning towards a combination of both adding a test for `ak.is_sorted` and always validating against NumPy. @pierce314159 what are your thoughts?

It looks like if we add the `NumPy` validation it will need to be done in all tests.

Reply:
I agree I think definitely validate `ak.is_sorted` is correct. I'm also down with always comparing against numpy

Reply:
I added `np_is_sorted` and `test_is_sorted` for review, but I'm not sure it does quite what is being asked.  I could not find a cleaner way to verify sortedness in numpy.

Reply:
I think the added test is achieving what we want on the testing of `ak.is_sorted()`. From a quick search the helper method does appear to be doing what is considered the accepted check within NumPy to determine if an array is sorted. I am going back and forth on whether or not we should  be using `np_is_sorted(ak_array.to_ndarray())` in place of `ak.is_sorted` for the other validations. I may be splitting hairs. @pierce314159 - what are your thoughts? I am torn between covering everything and saying that if we test `ak.is_sorted` it is acceptable to use for other validations.

Reply:
I get where you're coming from. I think if we did some edge_case testing with `is_sorted` I'd be comfortable only using that. But given https://github.com/Bears-R-Us/arkouda/issues/2703, I'm somewhat doubtful it would behave correctly. I think either is good for now and I'll revisit it during #2703

Reply:
I am fine with pushing this for now since we have an issue to revisit. I'll approve it. Thanks for weighing in @pierce314159.